docker/install_libs.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+
+LIBNAME="${1:-support}"
+LIVERSION="${2:-0.3.0}"
```

Comment:
I'd prefer `${1?}/${2?}` instead of a default value.

Reply:
Scratch that. I'd just pull up the check for 2 parameters to the top.

Reply:
done